src/node/markdown/plugins/preWrapper.ts

```diff
@@ -16,7 +17,12 @@ export function preWrapperPlugin(md: MarkdownItAsync, options: Options) {
     const [tokens, idx] = args
     const token = tokens[idx]
 
-    // remove title from info
+    // @ts-ignore
+    const isFromSnippet = !!token.src
```

Comment:
`src` here is [added by `snippet.ts`](https://github.com/vuejs/vitepress/blob/be260fda6efc1d6c4b56219d7a17a19ab7a4ba76/src/node/markdown/plugins/snippet.ts#L164-L165), however I think it's better if it's `token.meta.src` instead similar to how [`githubAlerts.ts` uses it](https://github.com/vuejs/vitepress/blob/be260fda6efc1d6c4b56219d7a17a19ab7a4ba76/src/node/markdown/plugins/githubAlerts.ts#L49-L52). But I didn't change it here as I don't want to accidentally break downstream if they use this internal info.